packages/perspective-viewer-d3fc/src/js/data/groupData.js

```diff
@@ -0,0 +1,51 @@
+/******************************************************************************
```

Comment:
I didn't expect to see another data function. Can we not use the same one as the line chart (splitData)?